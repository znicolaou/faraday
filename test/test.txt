0 0.001794 60.169048 1.984444 0.199925
1 0.002906 59.796664 0.865404 0.197799
2 0.002425 59.986483 3.185011 0.197861
2 -0.009776 9.732626 0.920998 -0.583002
