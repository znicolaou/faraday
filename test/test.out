accel.py --filebase test --delay 2 --Nt 150
